core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -251,7 +292,7 @@ func (i *pluginOracleCreator) createFactoryAndTransmitter(
 			publicConfig.DeltaRound,
 		)
 
-		rmnCrypto := ccipevm.NewEVMRMNCrypto(i.lggr.Named("EVMRMNCrypto"))
+		rmnCrypto := plugin.RMNCrypto(i.lggr.Named(chainFamily + ".RMNCrypto"))
```

Comment:
I'm partial to using structured logging fields for this sort of thing. i.e. `lggr.Named("RMNCrypto").With(chainFamily)`.

what do you think @mateusz-sekara @makramkd